In [1]:
import pandas as pd

path = r'../../Arquivos Csv\Base Completa\pns2019.csv'

df = pd.read_csv(path)

df.shape

(293726, 1087)

In [2]:
# Tipo de entrevista - apenas Realizada == 01
# apenas entre 30 e 60 anos
# Apenas quem respondeu Q074 - se possui asma ou não

df = df[df['V0015'] == 1]

print(df['V0015'].value_counts())

df = df[df['C008'].between(30, 60)]

print(df['C008'].value_counts(ascending=True))

print(df.shape)

df = df[((df['Q074'] == 1) | (df['Q074'] == 2))]

print(df['Q074'].value_counts())

print(df.shape)



V0015
1    279382
Name: count, dtype: int64
C008
60.0    2878
58.0    3067
57.0    3091
59.0    3093
56.0    3340
54.0    3486
53.0    3490
47.0    3495
51.0    3499
52.0    3587
55.0    3592
50.0    3625
48.0    3721
49.0    3726
46.0    3777
45.0    3811
44.0    3859
41.0    3905
43.0    3921
31.0    4029
42.0    4036
32.0    4065
33.0    4086
34.0    4088
30.0    4153
35.0    4155
38.0    4244
39.0    4264
40.0    4287
36.0    4329
37.0    4460
Name: count, dtype: int64
(117159, 1087)
Q074
2.0    49380
1.0     2450
Name: count, dtype: int64
(51830, 1087)


In [ ]:
df.to_csv('../../pns2019_Filtrada.csv')

# Numero de Registros

* Apenas entrevistas realizadas: **279382**.
* Apenas entrevistas realizas e entre 30 - 60 anos: **117159**.
* Apenas entrevistas realizadas, entre 30 - 60 anos e responderam se possuem asma ou não: **51830**.
